### PR TITLE
[docs] Cloud Billing

### PR DESCRIPTION
**Pull request by ddhodge**
Cloud billing and payment

### CODE REVIEW

**Comment by ddhodge** [comment]
Waiting on Invoicing piece from @gurushant-yb

**Comment by gurushant-yb** [comment]
> Waiting on Invoicing piece from @gurushant-yb

Hi @ddhodge,
I have added invoice listing and summary details, I will update usage details sometime by tomorrow.

**Review by lizayugabyte — COMMENTED** [review]
Comment on Administer Yugabyte Cloud page
_Before you can add paid clusters, you need to create a billing profile and add a payment method. The billing profile and payment methods are managed by the cloud account Admin user (the user who created the Yugabyte Cloud account) from the Admin page. The Admin user can also view your monthly invoices and invite other users to your cloud so that they can create, manage, authorize network access, and connect to clusters._

It's a bit confusing when the reader is referred to as "you" and then there's a switch to someone with an Admin account who can "view your monthly invoces ..."

File: docs/content/latest/yugabyte-cloud/cloud-admin/cloud-billing-profile.md [hunk 1]
```diff
@@ -70,9 +72,66 @@ The **Credits** section displays any credits applied to your account, including
...
+- Invoice Period - billing period for the invoice.
+- Subtotal - total invoiced amount to be paid.
+
+Select an invoice in the list to view the invoice **Summary**, which includes a detailed breakdown of costs and usage details. These details are also included on the invoice that is emailed to the billing address.
+
+To download the invoice as a PDF, click the **Download** icon.
+
+### Invoice summary
+
+To view invoice details, select an invoice in the **Invoice History** to display the **Summary**.
```
Comment (lizayugabyte) [inline docs/content/latest/yugabyte-cloud/cloud-admin/cloud-billing-profile.md:111]: To view invoice details, select an invoice in the **Invoice History** to display the **Summary**. -- I'm not sure "the"s are needed here. I'd say it should be either 
To view invoice details, select an invoice in **Invoice History** to display **Summary**.
or
To view invoice details, select an invoice in the **Invoice History** section to display the **Summary** section.
or
To view invoice details, select an invoice by navigating to **Invoice History > Summary**.

File: docs/content/latest/yugabyte-cloud/cloud-admin/cloud-billing-profile.md [hunk 2]
```diff
@@ -70,9 +72,66 @@ The **Credits** section displays any credits applied to your account, including
...
+
+To download the invoice as a PDF, click **Download Invoice**.
+
+The **Summary** shows the following invoice details:
```
Comment (lizayugabyte) [inline docs/content/latest/yugabyte-cloud/cloud-admin/cloud-billing-profile.md:115]: The **Summary** shows the following invoice details:  --> **Summary** shows the following invoice details:

File: docs/content/latest/yugabyte-cloud/cloud-admin/cloud-billing-profile.md [hunk 3]
```diff
@@ -70,9 +72,66 @@ The **Credits** section displays any credits applied to your account, including
...
+- Billing option - your billing plan (pay-as-you-go or subscription).
+- Billing period - the start and end date of the current billing period. Your first billing period starts the day you created your billing profile and ends on the last day of the month; subsequent periods start on the first day of the month and end on the last.
+- Previous invoice - the amount of the previous invoice.
+- Last billed on - the date of the previous invoice.
+- This time last month - the amount owing on the same day in the previous billing period (for example, if the current day is the 13th, this displays the amount that was owed on the 13th of last month.
+- Payment status - whether the invoice was paid.
+- Payment date - the date the invoice was paid.
+- Payment method - the payment method used to pay the invoice.
+
+In addition, the **Summary by Usage** shows a breakdown of costs for each dimension, including:
```
Comment (lizayugabyte) [inline docs/content/latest/yugabyte-cloud/cloud-admin/cloud-billing-profile.md:127]: Same comment

File: docs/content/latest/yugabyte-cloud/cloud-basics/connect/ycql.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ To connect to a cluster using `ycqlsh`:
 1. Click **Connect**.
 1. Click **YugabyteDB Client Shell**.
 1. If you have not installed the client shell on your computer, select your operating system and copy the command to install the shell from the command line.
-1. Click **Download CA Cert** to download the root.crt certificate for TLS encryption, and install the certificate on your computer.
+1. Click **Download CA Cert** to download the root.crt certificate for TLS encryption, and install the certificate on your computer. If you are using Docker, copy it to your Docker container.
```
Comment (lizayugabyte) [inline docs/content/latest/yugabyte-cloud/cloud-basics/connect/ycql.md:7]: If you are using Docker, copy it to your Docker container.  -->  Reads like "it" refers to Docker. Should be changed to cert

File: docs/content/latest/yugabyte-cloud/cloud-basics/connect/ysql.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ To connect to a cluster using `ysqlsh`:
 1. Click **Connect**.
 1. Click **YugabyteDB Client Shell**.
 1. If you have not installed the client shell on your computer, select your operating system and copy the command to install the shell from the command line.
-1. Click **Download CA Cert** to download the root.crt certificate for TLS encryption, and install the certificate on your computer.
+1. Click **Download CA Cert** to download the root.crt certificate for TLS encryption, and install the certificate on your computer. If you are using Docker, copy it to your Docker container.
```
Comment (lizayugabyte) [inline docs/content/latest/yugabyte-cloud/cloud-basics/connect/ysql.md:7]: Same comment about "it"

**Review by lizayugabyte — APPROVED** [review]
Minor comments. Otherwise LGTM

**Pull request merged**